public/app/features/browse-dashboards/components/BrowseFilters.tsx

```diff
@@ -10,7 +10,6 @@ export function BrowseFilters() {
   return (
     <div>
       <ActionRow
-        hideLayout
```

Comment:
i know i've asked this before and i can't quite remember the response... are we expecting to be able to remove this when `nestedFolders` GAs or not? 🤔 if so, we could do something like `hideLayout={config.featureToggles.isEnabled('nestedFolders')}`. if not, disregard this! 😅

Reply:
I think we are not going to remove it but I might be wrong. @joshhunt can you tell us the answer? We will remember it as it will be written down 😅